src/com/hp/ppm/integration/jira/JIRAWorkPlanIntegration.java

```diff
@@ -0,0 +1,163 @@
+package com.hp.ppm.integration.jira;
```

Comment:
As you may know, HP Software will soon be sold to a company called Micro Focus. In the past, the name of the company owning PPM product already change 4 times (Chainlink / Kintana / Mercury / HP / HPE).
So I think it's better if we don't use "HP" nor any other company name in the package :)

For the sample connector, I used package
com\ppm\integration\agilesdk\connector\sample

so maybe for this one you could use 
com\ppm\integration\agilesdk\connector\jira